Chetna_Mongmaw/calculator/assets/scripts/calc.js

```diff
@@ -0,0 +1,29 @@
+function display(val){
+    document.getElementById("result").value+=val ;
+}
+function compute(){
+    let x = document.getElementById("result").value;
+    
+    x = x.replace(/sin/g, "Math.sin");
+    x = x.replace(/cos/g, "Math.cos");
+    x = x.replace(/tan/g, "Math.tan");
+    x = x.replace(/√/g, "Math.sqrt")
+    x = x.replace(/x/g, "*")
+    x = x.replace(/÷/g, "/")
+
+
+
+    let y = eval(x) 
+    document.getElementById("result").value = y 
+}
+
+function cscreen() 
```

Comment:
Indentation missing
naming convention missing
empty function not removed
function comments to brief about its purpose missing